lessons/Early-Math/identity-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,108 @@
+= The Identity Property
+
+@description{This lesson introduces students to the Identity Property by analyzing and exploring Circles of Evaluation.}
+
+@lesson-prereqs{equivalence}
+
+@keywords{Circle of Evaluation, Identity Property}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize that multiplying and dividing by one does not affect the value of an expression.
+* Recognize that adding or subtracting zero does not affect the value of an expression.
+* Demonstrate their understanding of the Identity Property with and without variables via Circles of Evaluation, numeric expressions, and words.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Identity Property and Circles of Evaluation!
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+== The Identity Property
+
+=== Overview
+
+Students analyze pairs of Circles to see if they represent equivalent expressions. The Circles make visible the structural changes that occur when we apply the Identity Property.
+
+=== Launch
+
+Build on prior knowledge by reminding students of the activity that they completed in the previous lesson. You might even refer to the list of category names that the class created, and then draw students’ attention to any category names that allude to or suggest the commutativity. (Any mention of “changing the order” is a strong hint that students’ have clued into the Commutative Property!)
```

Comment:
guessing you meant to give a clue for the Identity property rather than the commutative property here?